backend/src/DataSources/posts.spec.js

```diff
@@ -149,7 +162,9 @@ describe("queries", () => {
 });
 
 describe("mutations", () => {
-
+    beforeEach(() => {
+        server.context = () => ({decodedJwt: { id: usersMemory.users[1].id}})//new User({name:'Ilona', email:'ilona@gmail.com', password:'12345678'})
```

Comment:
I like to have hard-coded values here:
```suggestion
        server.context = () => ({decodedJwt: { id: "42"}})//new User({name:'Ilona', email:'ilona@gmail.com', password:'12345678'})
```
It's less error-prone and avoid false-negatives. Searching for the string `42` within this file is totally acceptable and probably you never have to refactor it, so redundancy doesn't hurt here.